rita/src/rita_common/tunnel_manager/mod.rs

```diff
@@ -288,6 +291,49 @@ impl Handler<GetNeighbors> for TunnelManager {
     }
 }
 
+/// A message type for deleting all tunnels we haven't heard from for more than the duration.
+pub struct TriggerGC(pub Duration);
```

Comment:
Obligatory tuple vs struct readability nitpick: lets add a name for duration?

Reply:
we only have one object here. I think that's too far.